packages/core/src/lib/implementation/persist.ts

```diff
@@ -29,27 +29,34 @@ export async function persistReport(
 ): Promise<MultipleFileResults> {
   const { outputDir, filename, format } = options;
 
-  let scoredReport = scoreReport(report);
-  console.info(reportToStdout(scoredReport));
+  let scoredReport;
```

Comment:
I just wonder about this place:
We have the task #352, which is about the report sorting on the earlier stage.
Shouldn't we keep assigning of `scoredReport` here, and sort it on the next string (in the scope of another task ofc)?
This way the sorted `scoredReport` could be passed to each necessary format.

Reply:
I assign it only if needed. Should be below in the code. And should be shared. No?